### ISSUE TITLE
Lonely "next" button in list view

### ISSUE THREAD

**Issue by dellermann** [dellermann/springcrm#5]
In some list views, where all available items fit one page, a lonely "next" button from the paginator is displayed (see screenshot).

![screenshot-purchase-invoice-list](https://f.cloud.github.com/assets/1315989/1606484/45b82b1e-5443-11e3-844e-b9bab42fda58.png)

_Related to Bugzilla bug 193._

**Comment by dellermann** [comment]
The error seems to be caused by the `<g:paginate>` tag. We should analyze this problem deeper, especially after release of Grails 2.0.

**Comment by dellermann** [comment]
The bug seems to occur if there is only one entry in the list.

**Comment by dellermann** [comment]
The "next" button disappears after restarting the application.

**Comment by dellermann** [comment]
In SpringCRM 2.0, the error still occurs, but the styles are different (see screenshot):

![image](https://cloud.githubusercontent.com/assets/1315989/13123794/464a20d8-d5be-11e5-8814-28973a2103fd.png)

In source code I found the following HTML snippet:

``` html
      <nav class="text-center">
        <div class="visible-xs">
          <li role="listitem" aria-label="Nächste"><a href="/organization/index?listType=&amp;offset=0&amp;max=10" class="nextLink"><span aria-hidden="true">»</span></a></li>
        </div>
        <div class="hidden-xs">
          <li role="listitem" aria-label="Nächste"><a href="/organization/index?listType=&amp;offset=0&amp;max=10" class="nextLink"><span aria-hidden="true">»</span></a></li>
        </div>
      </nav>
```

**Comment by dellermann** [comment]
The issue occurs only after creating the first item in a list. I investigate in that issue and found out that a negative offset value is stored in the session.

**Comment by dellermann** [comment]
Fixed in commit 924d0db.
